fr/reference/tutorial-rest.rst

```diff
@@ -117,9 +117,9 @@ Quand une requête URI corresponds à une route défini, l'application exécute
 
 Créer un Model
 --------------
-Notre API fourni des informations sur les 'robots', ces données doivent donc être enregistrés dans une base de données.
+Notre API fournit des informations sur les 'robots', ces données doivent donc être enregistrées dans une base de données.
 Le model suivant nous permet d'accéder à la table comme si c'était un objet. Nous avons implémenté quelques règles en utilisant
-des validateurs. Ainsi nous seront tranquille car les données respecteront toujours les conditions nécessaire pour notre application:
+des validateurs. Ainsi nous serons tranquilles car les données respecteront toujours les conditions nécessaires pour notre application:
```

Comment:
nous => respecteronS
ils/elles => respecteronT

les données = elles, donc respecteronT


y'a pas de soucis, si je trouve le temps, j'essaierai de passer sur un max de la doc française pour proposer des corrections ;-)


Cordialement,

Samuel,
Sam007@free.fr

----- Mail original -----
De: "Sid Roberts" <notifications@github.com>
À: "phalcon/docs" <docs@noreply.github.com>
Cc: "Samuel Thery" <sam007@free.fr>
Envoyé: Jeudi 29 Octobre 2015 12:17:49
Objet: Re: [docs] Update tutorial-rest.rst (#726)



In fr/reference/tutorial-rest.rst : >  Le model suivant nous permet d'accéder à la table comme si c'était un objet. Nous avons implémenté quelques règles en utilisant
> -des validateurs. Ainsi nous seront tranquille car les données respecteront toujours les conditions nécessaire pour notre application:
> +des validateurs. Ainsi nous serons tranquilles car les données respecteront toujours les conditions nécessaires pour notre application: 

Mon français n'est pas bon. ;) Devrait-il être «respecteront» ou «respectons»? 

— 
Reply to this email directly or view it on GitHub .